src/EntryPoints/REST/ApprovePageApi.php

```diff
@@ -75,10 +78,16 @@ public function newSuccessResponse( ?ApprovalState $state ): Response {
 			return $this->getResponseFactory()->createNoContent();
 		}
 
+		$language = $this->languageFactory->getLanguage( $this->getRequest()->getHeaderLine( 'Accept-Language' ) );
```

Comment:
This can't be the best way to get at the language

Reply:
`\RequestContext::getMain()->getLanguage()` seems better, though it's be nicer to also get it from something given to this class, rather than from global state